scripts/publish-image/build-publish-image.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if ! type docker > /dev/null; then
+  exit 1
+fi
+
+: "${REGISTRY:=${REGISTRY}}"
+: "${SDK_IMG_NAME:=carbon-aware}"
+: "${SDK_TAG:=${SDK_TAG}}"
+
+error() {
+    echo "$@" >&2
+    exit 1
+}
+
+OUTPUT_TYPE=type=registry
+BUILDPLATFORM=linux/amd64
```

Comment:
Can this also be built for `linux/arm64`? I'm pretty sure it can just be appended as a comma-separated list.

Reply:
This is only the default value for the pipeline. The value Can be set via Dockerfile. I'll add it now.